src/org/usfirst/frc/team2473/robot/Robot.java

```diff
@@ -0,0 +1,111 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Scheduler;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import org.usfirst.frc.team2473.framework.Devices;
+import org.usfirst.frc.team2473.robot.commands.DriveForDistanceCommand;
+import org.usfirst.frc.team2473.robot.commands.PointTurn;
+import org.usfirst.frc.team2473.robot.subsystems.DrivePower;
+import org.usfirst.frc.team2473.robot.subsystems.DriveSubsystem;
+
+public class Robot extends TimedRobot {
```

Comment:
`TimedRobot` calls periodic methods at a 20 millisecond interval while `IterativeRobot` calls them upon arrival of new data from the Driver Station. `IterativeRobot` calls at a period of approximately 20 milliseconds too but it can vary depending on the conditions. `TimedRobot` would be a better choice for us because our control algorithms work better at a set periodic interval.

Reply:
There's a handy comparison of base classes [here](https://wpilib.screenstepslive.com/s/currentCS/m/cpp/l/241853-choosing-a-base-class).